packages/frontend/components/Video.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import ReactPlayer from 'react-player'
+import styles from '../styles/videoBoard.module.scss'
+
+export default function Video({ video }) {
```

Comment:
This is also the same as above review, please add the type for `video`, for example you can create an interface for it
If you are unfamiliar with interface here I provide you some simple explanation of it https://www.tutorialspoint.com/typescript/typescript_interfaces.htm

Reply:
fixed